cli/compose/loader/loader_test.go

```diff
@@ -825,3 +825,31 @@ networks:
 
 	assert.Equal(t, expected, config.Networks)
 }
+
+func TestLoadExpandedMountFormat(t *testing.T) {
+	config, err := loadYAML(`
+version: "3.1"
+services:
+  web:
+    image: busybox
+    volumes:
```

Comment:
Yes, but it's also a breaking change, so not something we can change in the `v3.1` format. We missed our chance with v3, so we can queue it up for v4 I guess (depending on the outcome of the `docker run` flag change).